state.go

```diff
@@ -694,6 +694,9 @@ func (m *Memberlist) aliveNode(a *alive, notify chan struct{}, bootstrap bool) {
 		}
 	}
 
+	// Clear out any suspicion timer that may be in effect.
+	delete(m.nodeTimers, a.Node)
```

Comment:
We should probably move this below all the checks of incarnation number to avoid an old packet from causing the timer to get reset